recipes/rhessys/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "7.2" %}
+
+package:
+  name: rhessys
+  version: {{ version }}
+
+source:
```

Comment:
We'd generally prefer for you to use a tarball release rather than a git revision:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
I used a tarball release.

Reply:
This still looks like it is pulling a git repo via `git_url` and the revision rather than  just using `url` with a source distribution tarball.

Reply:
See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L19